includes/controllers/pages/class-checkout.php

```diff
@@ -175,7 +175,11 @@ private function checkout_form() {
         // $checkout_form .= wpbdp_capture_action( 'wpbdp_checkout_form_top', $this->payment );
         $checkout_form .= $this->gateway->render_form( $this->payment, $this->errors );
         // $checkout_form .= wpbdp_capture_action( 'wpbdp_checkout_form_bottom', $this->payment );
-        $checkout_form .= '<div class="wpbdp-checkout-submit"><input type="submit" value="' . _x( 'Pay Now', 'checkout', 'business-directory-plugin' ) . '" /></div>';
+        $pay_now = ( $this->payment->has_item_type( 'recurring_plan' ) || $this->payment->amount > 0 );
+        $checkout_form .= sprintf(
+            '<div class="wpbdp-checkout-submit"><input type="submit" value="%s" /></div>',
+            $this->payment->show_payment_options() ? _x( 'Pay Now', 'checkout', 'business-directory-plugin' ) : _x( 'Complete', 'checkout', 'business-directory-plugin' )
```

Comment:
This is missing sanitizing

Reply:
Not sure about what should be sanitized here, but text has been changed to use `esc_html__`

Reply:
Yep, exactly. The translations are output that should always be sanitized too.